app/src/main/java/io/github/droidkaigi/confsched/prefs/DefaultPrefsSchema.java

```diff
@@ -0,0 +1,27 @@
+package io.github.droidkaigi.confsched.prefs;
+
+import android.content.Context;
+
+import com.rejasupotaro.android.kvs.annotations.Key;
+import com.rejasupotaro.android.kvs.annotations.Table;
+
+@Table("io.github.droidkaigi.confsched_preferences")
+public class DefaultPrefsSchema {
```

Comment:
Good point! I updated it in https://github.com/rejasupotaro/droidkaigi2016/commit/655310dd4573ee9ee94b22afe0051595553dbcd8